internal/constants/constants.go

```diff
@@ -2,3 +2,90 @@ package constants
 
 const SNYK_DEFAULT_API_URL = "https://api.snyk.io"
 const SNYK_DEFAULT_API_VERSION = "2024-04-22"
+
+var PRODUCTS = []string{"code", "iac", "container", "sbom"}
+var KNOWN_COMMANDS = []string{"test", "code", "monitor", "iac", "rules", "describe", "sbom", "container", "debug"}
+var WHITELISTED_FLAGS = []string{
```

Comment:
**suggestion (blocking):** Please lets switch to ALLOWED_FLAGS here. It clearly articulates the meaning.  

Supporting material: https://www.ncsc.gov.uk/blog-post/terminology-its-not-black-and-white

Reply:
In addition, I would move this to the `categories.go` file.